src/TwigFakerExtension.php

```diff
@@ -89,7 +90,6 @@ private function getCachedData($type, $count, $cache_key)
     private function createNewFakeData($type, $count)
     {
         $faker = Factory::create();
-        $this->fake_data = [];
```

Comment:
since the array is cleard in the fakerData methos, it is not needed here :smile: